lib/administrate/base_dashboard.rb

```diff
@@ -88,7 +88,7 @@ def attribute_not_found_message(attr)
 
     def attribute_includes(attributes)
       attributes.map do |key|
-        field = self.class::ATTRIBUTE_TYPES[key]
+        field = self.class::ATTRIBUTE_TYPES.fetch(key)
```

Comment:
Good call; thank you. Since we are at it, how about using `attribute_type_for` instead? That way we also get a more descriptive error than just a `KeyError`.

Reply:
Ok, I didn't know what is `attribute_type_for`

Now googled and found that it is an [internal administrate method](https://www.rubydoc.info/gems/semantic-administrate/0.1.0/Administrate/BaseDashboard:attribute_type_for). 

[Kind of done](https://github.com/thoughtbot/administrate/pull/1762/commits/0a050c9014a8ee423bbd528eb8ec0604223f72db), I guess.

Reply:
Ok, blind commit was not a good idea